weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -250,6 +250,10 @@ def call_start(self, req: tsi.CallStartReq) -> tsi.CallStartRes:
             trace_id=ch_call.trace_id,
         )
 
+    # TODO: This is not actually implemented at the lower level yet.
+    def call_start_batch(self, req: tsi.CallCreateBatchReq) -> tsi.CallCreateBatchRes:
+        return tsi.CallCreateBatchRes(res=[])
```

Comment:
is this safe?

Reply:
why not actually implement this?

Reply:
We could!  But this slice of the stack was already getting big and I didn't want to implement batching for all of the servers in this same PR.